data_adapter_oemof/build_datapackage.py

```diff
@@ -163,7 +156,13 @@ def build_datapackage(cls, adapter: Adapter):
                 parametrized_elements[facade_adapter_name] = scalars
 
             parametrized_sequences = {process_name: timeseries}
-        parametrized_elements = add_bus_to_element_dict(parametrized_elements)
+        parametrized_elements["bus"] = pd.DataFrame(
+            {
+                "name": pd.unique(parametrized_elements["bus"]),
+                "type": ["bus" for i in pd.unique(parametrized_elements["bus"])],
+                "blanced": [True for i in pd.unique(parametrized_elements["bus"])],
```

Comment:
You are calling `pd.unique` three times (two times you only need length of it).
Maybe you could use [walrus operator `:=`](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions) here ;)

Reply:
Using walrus in 6b0f23df5c56ac66ddb55313ef3566328cba23da always learning, thank you :+1: